runners_manager/runner/RunnerManager.py

```diff
@@ -65,6 +65,13 @@ def update(self, github_runners: list[dict]):
             while self.need_new_runner(vm_type):
                 self.create_runner(vm_type)
 
+            # Respawn runner if they are offline for more then 10min after spawn
+            for elem in self.filter_runners(vm_type, lambda r: r.status == 'offline'):
+                time_since_spawn = datetime.datetime.now() - elem.created_at
+                if elem.has_run is False \
+                        and time_since_spawn > datetime.timedelta(minutes=15):
```

Comment:
This setting is pretty cool. we could add a TODO here to make it configurable one day. Maybe, maybe not :D